src/SprykerEco/Zed/Optivo/Business/OptivoFacadeInterface.php

```diff
@@ -1,45 +1,84 @@
 <?php
 
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
 namespace SprykerEco\Zed\Optivo\Business;
 
 use Generated\Shared\Transfer\MailTransfer;
-use Generated\Shared\Transfer\OptivoSubscribeRequestTransfer;
-use Generated\Shared\Transfer\OptivoTransactionalMailRequestTransfer;
-use Generated\Shared\Transfer\OptivoUnsubscribeRequestTransfer;
 
 interface OptivoFacadeInterface
 {
     /**
      * Specification:
-     * - Receives the fully configured MailTransfer
-     * - Sends the mail
+     *  - This method is used by OptivoNewOrderPlugin Oms command for sending data to Optivo API
      *
      * @api
      *
-     * @param \Generated\Shared\Transfer\MailTransfer $mailTransfer
+     * @param int $idSalesOrder
+     *
+     * @return void
+     */
+    public function handleNewOrderEvent(int $idSalesOrder): void;
+
+    /**
+     * Specification:
+     *  - This method is used by OptivoOrderCanceledPlugin Oms command for sending data to Optivo API
+     *
+     * @api
+     *
+     * @param int $idSalesOrder
+     *
+     * @return void
+     */
+    public function handleOrderCanceledEvent(int $idSalesOrder): void;
+
+    /**
+     * Specification:
+     *  - This method is used by OptivoPaymentNotReceivedPlugin Oms command for sending data to Optivo API
+     *
+     * @api
+     *
+     * @param int $idSalesOrder
      *
      * @return void
      */
-    public function sendMail(MailTransfer $mailTransfer);
+    public function handlePaymentNotReceivedEvent(int $idSalesOrder): void;
 
     /**
-     * @param \Generated\Shared\Transfer\OptivoSubscribeRequestTransfer $optivoSubscribeRequestTransfer
+     * Specification:
+     *  - This method is used by OptivoShippingConfirmationPlugin Oms command for sending data to Optivo API
+     *
+     * @api
+     *
+     * @param int $idSalesOrder
      *
-     * @return \Generated\Shared\Transfer\OptivoResponseTransfer
+     * @return void
      */
-    public function sendSubscribeRequest(OptivoSubscribeRequestTransfer $optivoSubscribeRequestTransfer);
+    public function handleShippingConfirmationEvent(int $idSalesOrder): void;
 
     /**
-     * @param \Generated\Shared\Transfer\OptivoUnsubscribeRequestTransfer $optivoUnsubscribeRequestTransfer
+     * Specification:
+     *  - Handle customer event
```

Comment:
Please add more meaningful specification like: sends request, saves to DB, updates something and so on.

Reply:
Could you, please, show en example?
Cannot understand what is wrong here